examples/basics/basics.ipynb

```diff
@@ -1,441 +1,468 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    from labelbox.schema.media_type import MediaType
</pre><p>It would be nice to add these deps to the top level namespace.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/745/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I can do this moving forward , however, I am going to delete this two imports since I don't really use them on this notebook </p>